Resources/Prototypes/_RMC14/Roles/Jobs/Other/PVE/RCM/troop_commander.yml

```
@@ -0,0 +1,62 @@
+- type: job
+  parent: CMJobBase
+  id: CMRoyalMarinesTroopCommanderPVE
+  name: rmc-job-name-rcm-pve-troop-commander
+  description: rmc-ghost-role-information-rcm-description
+  playTimeTracker: RMCJobRoyalTroopCommanderPVE
+  ranks:
+    RMCRankTSESecondLieutenant: [ ]
+  startingGear: RMCGearRoyalTroopCommanderPVE
+  icon: "RMCJobIconRoyalTroopCommanderPVE"
+  joinNotifyCrew: false
+  supervisors: cm-job-supervisors-royal-captain
+  spawnMenuRoleName: RCM Troop Commander (PVE)
+  special:
+  - !type:AddComponentSpecial
+    components:
+    - type: NpcFactionMember
+      factions:
+      - RoyalMarines
+      - TSE
+    - type: Skills
+      skills:
+        RMCSkillCqc: 3
+        RMCSkillFirearms: 1 
+        RMCSkillFireman: 3
+        RMCSkillConstruction: 1
+        RMCSkillEngineer: 2
+        RMCSkillEndurance: 3
+        RMCSkillMedical: 2
+        RMCSkillSurgery: 1  
```

This needs surgery 2 to be inline with the other platco roles.
```suggestion
        RMCSkillSurgery: 2
```